app/resources/cdl/event_logging.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module CDL
+  class EventLogging
+    class << self
+      def get_patron_group(netid:)
+        return false unless netid
+        patron_record = Faraday.get "#{CDL::EligibleItemService.bibdata_base}patron/#{netid}"
+        if patron_record.status == 200
+          patron = JSON.parse(patron_record.body).with_indifferent_access
+          patron["patron_group"]
+        end
+      rescue
+        false
```

Comment:
I haven't seen the coverage yet, but I think this line is still uncovered. Also, the rescue should also return `nil` as per the previous conversation.

Reply:
the line is covered but for `false.` I will update for nil